pkg/kind/kind.go

```diff
@@ -110,6 +115,23 @@ func checkDocker() error {
 	return nil
 }
 
+func createLocalRegistry() error {
+	fmt.Println("./hack/kind/local-registry.sh", container_reg_name, container_reg_port)
+	localRegCheck := exec.Command("./hack/kind/local-registry.sh", container_reg_name, container_reg_port)
+	if err := localRegCheck.Run(); err != nil {
+		return fmt.Errorf("failed to create local registry container: %s", err.Error())
+	}
+	return nil
+}
+
+func connectLocalRegistry() error {
+	connectLocalRegistry := exec.Command("./hack/kind/connect-local-registry.sh", container_reg_name, container_reg_port)
+	if err := connectLocalRegistry.Run(); err != nil {
+		return fmt.Errorf("failed to connect local registry to kind cluster")
+	}
+	return nil
+}
```

Comment:
Rather than running these as scripts, can we call them directly from `pkg/kind/kind.go`, similar to how we do with `createNewCluster()`?

Reply:
Yeah, I make it.